src/promdens/promdens.py

```diff
@@ -93,15 +93,13 @@ class LaserPulse:
     equation: str = dataclasses.field(init=False)
 
     def __str__(self):
-        return (
-            f"Laser pulse parameters:\n"
-            f"  E(t) = {self.equation}\n"
-            f"  - envelope type = '{self.envelope_type}'\n"
-            f"  - omega = {self.omega:.6f} a.u.\n"
-            f"  - FWHM = {self.fwhm:.6f} a.u.\n"
-            f"  - t0 = {self.t0:.6f} a.u.\n"
-            f"  - chirp = {self.lchirp:.3e} a.u."
-        )
+        return (f"Laser pulse parameters:\n"
```

Comment:
It's not great that we keep going back and forth on the formatting. I am going to put up a PR with setting up a ruff formatter if you don't mind (can change it if you like some other formatter better)

Reply:
Sounds like a good plan. Let's unify the formatting as you propose.